regexes/client/libraries.yml

```diff
@@ -566,3 +566,18 @@
   name: 'request'
   version: ''
   url: 'https://github.com/request/request'
+
+- regex: 'kiwi-tcms/([a-z\d\.A-Z]+)'
+  name: 'Kiwi TCMS'
+  version: '$1'
+  url: 'https://kiwitcms.org'
+
+- regex: 'tcms-api/([a-z\d\.A-Z]+)/Python(.*)'
```

Comment:
```suggestion
- regex: 'tcms-api/(\d+[\.\d]+)'
```

Reply:
You have to add a test user-agent to `Tests/Parser/Client/fixtures/library.yml`.